src/localization/relocalizers/lsh_cv_hashing.cpp

```diff
@@ -60,16 +60,16 @@ void LshCvHashing::train(std::vector<iFeature::Ptr> features) {
   // m_matcherPtr->match(*query, knnmatches, 1);
 }
 
-std::vector<int> LshCvHashing::hashFeature(const iFeature::ConstPtr &fPtr) {
+std::vector<int> LshCvHashing::hashFeature(const iFeature &feature) {
   std::vector<std::vector<cv::DMatch>> matches;
-  cv::Mat feature(1, fPtr->bits.size(), CV_8UC1);
-  for (int i = 0; i < fPtr->bits.size(); ++i) {
-    feature.at<uchar>(0, i) = fPtr->bits[i];
+  cv::Mat featureCV(1, feature.bits.size(), CV_8UC1);
+  for (int i = 0; i < feature.bits.size(); ++i) {
+    featureCV.at<uchar>(0, i) = feature.bits[i];
   }
   // _matcherPtr->match(feature, matches);
   Timer timer;
   timer.start();
-  _matcherPtr->knnMatch(feature, matches, 5);
+  _matcherPtr->knnMatch(featureCV, matches, 5);
```

Comment:
nit: starting variables with an `_` is usually reserved for the STL code. I would probably move the `_` to the end of the variable.

Reply:
Yeap, this is going to be changed in the next iterations when I am going to refactor the relocalizers.